homeassistant/components/device_tracker/icloud.py

```diff
@@ -0,0 +1,134 @@
+"""
+homeassistant.components.device_tracker.icloud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Device tracker platform that supports scanning a iCloud devices.
+
+It does require that your device has registered with Find My iPhone.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/device_tracker.icloud/
+"""
+import logging
+from datetime import timedelta
+import threading
+
+from homeassistant.const import CONF_USERNAME, CONF_PASSWORD
+from homeassistant.helpers import validate_config
+from homeassistant.util import Throttle
+from homeassistant.components.device_tracker import DOMAIN
+
+import re
+
+# Return cached results if last scan was less then this time ago
+MIN_TIME_BETWEEN_SCANS = timedelta(seconds=60)
+
+_LOGGER = logging.getLogger(__name__)
+
+REQUIREMENTS = ['https://github.com/picklepete/pyicloud/archive/'
```

Comment:
Because they have a pull request that allows the `verify=True` which will stop the InsecureWarnings from urrllib3 from happening. That settings allows a secure connection. The merge failed tests and a new release has not been put out yet, which is why we are specifying the commit of the merged pull request.